runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -4429,6 +4429,163 @@ void JitShutdown(J9JITConfig * jitConfig)
    TRC_JIT_ShutDownEnd(vmThread, "end of JitShutdown function");
    }
 
+#if defined (LINUX)
+//   Since Linux 4.0 only users with the CAP_SYS_ADMIN capability can get PFNs.
+//   In 4.0 and 4.1 opens by unprivileged fail with -EPERM.  Starting from
+//   4.2 the PFN field is zeroed if the user does not have CAP_SYS_ADMIN.
+//   Reason: information about PFNs helps in exploiting Rowhammer vulnerability.
+typedef struct __attribute__ ((__packed__))
```

Comment:
Probably worth putting a link to https://man7.org/linux/man-pages/man5/proc.5.html in the comment to specify what this struct is supposed to represent.

Reply:
Added link to another resource